packages/clerk-js/src/utils/__tests__/url.test.ts

```diff
@@ -84,18 +84,38 @@ describe('isValidUrl(url,base)', () => {
   });
 });
 
-describe('isRelativeUrl(url,base)', () => {
+describe('isProblematicUrl(url)', () => {
```

Comment:
Could potentially do with more test cases here, but with the shift to checking against full URLs we get more guarantees from the existing origin check.